README.md

```diff
@@ -22,7 +22,7 @@ The `setup_for_wrf.py` script does the following:
 * Performs substitutions of the config file. For example, if used, the shell environment variable `${HOME}` will be replaced by its value when interpreting the script. The variable `wps_dir` is defined within the config file, and if the token `${wps_dir}` appears within the configuration entries, such tokens will be replaced by the value of this variable.
 * Configure the main coordination script
 * Loop over the required WRF jobs, performing the following:
-  * Check if the WRF input files for this run are available (`wrfinput_d0?`). If not, perform the following:
+  * Check if the WRF input files for this run are available (`wrfinput_d01`). If not, perform the following:
     * Check that the geogrid files are available (copies should be found in the directory given by the config variable `nml_dir`). If not available, configure the WPS namelist and run `geogrid.exe` to produce them.
```

Comment:
What's WPS? I couldn't find the full name anywhere

Reply:
`WRF Pre-Processing System`. Nested acronyms!!!

WPS prepares the input data for a WRF run:

```
Main programs: geogrid.exe, ungrib.exe, metgrid.exe

Input to the main programs is through the namelist
file "namelist.wps".  Each main program has an 
exclusive namelist record (named "geogrid", "ungrib",
or "metgrid", respectively), and the three programs 
have a group record (named "share") that each program
reads.

geogrid
-------
1) Defines the model horizontal domain
2) Horizontally interpolates static data to the model
domain
3) Output conforms to the WRF I/O API

ungrib
------
1) Decodes Grib Edition 1 and 2 data
2) Uses tables to decide which variables to extract
3) Supports isobaric and generalized vertical coordinates
4) Output is in a non-WRF-I/O-API form, referred to as an
   intermediate format

metgrid
-------
1) Ingest static data and raw meteorological fields
2) Horizontally interpolate meteorological fields to the 
   model domain
3) Output conforms to WRF I/O API
```
https://github.com/wrf-model/WPS